sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -60,6 +60,7 @@
     private static final String REQUEST_RESPONSE_MESSAGES = "messages";
     private static final String REQUEST_RESPONSE_MESSAGE = "message";
     private static final String REQUEST_RESPONSE_EXPIRATIONS = "expirations";
+    private static final String LOCK_TOKEN_KEY = "lock-token";
```

Comment:
This exists in ManagementConstants

Reply:
All the constants in `ManagementConstants`  are package-private  and serializer can not access them. Unless we want to change that and make them public. Thus I added it here  in serializer.